src/ConnectionFactory.php

```diff
@@ -213,22 +210,18 @@ private function addDatabaseSuffix(array $params): array
      * @param mixed[] $params The list of parameters.
      * @phpstan-param Params $params
      *
-     * @return mixed[] A modified list of parameters with info from a database
+     * @return Params params A modified list of parameters with info from a database
```

Comment:
I'm not sure of this `Params`, it is a phpstan type not a native php or bundle type. The `@phpstan-return Params` is present bellow